File: MPI/mpicheck/tests/functions/1.0/MPI_Startall.c
#include <mpi.h>
int main(char argc, char**argv)
{
    /* vars */
    int var_0;
    MPI_Request var_1[2];
    int ret;
    /* calls */
    ret = MPI_Startall(var_0, var_1);
    ret = PMPI_Startall(var_0, var_1);
    return 0;
}

File: MPI/mpicheck/tests/functions/1.0/MPI_Startall.f

        
        program main
        include 'mpif.h'
        
        INTEGER var_0
       INTEGER var_1(10)
       INTEGER var_2
        call mpi_startall(var_0, var_1, var_2)
       call pmpi_startall(var_0, var_1, var_2)
        end program main
    